functional-tests/src/test/resources/arquillian.xml

```diff
@@ -12,7 +12,7 @@
       <property name="env.setup.script.url">setup.sh</property>
       <property name="env.teardown.script.url">destroy.sh</property>
       <property name="namespace.use.current">true</property>
-      <property name="wait.for.service.list">caching-service-app-hotrod, shared-memory-service-app-hotrod</property>
+      <property name="wait.for.service.list">caching-service-app-hotrod, shared-memory-service-app-hotrod, caching-service-app-http, shared-memory-service-app-http</property>
```

Comment:
Is it needed? Is so, there must be a bug in Arquillian since the same set of pods is behind each of those services.

Reply:
Good spot, I added this when I was running into some problems and it seemed to solve them but I will double check